### PR TITLE
Use fnv64 for InmemCacheService

### PR DESCRIPTION
**Pull request by grobinson-grafana**
**Which issue(s) does this PR fix?**:

Use `fnv64` hash of the screenshot options instead of `fmt`.

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/services/screenshot/option.go [hunk 1]
```diff
@@ -41,3 +43,14 @@ func (s ScreenshotOptions) SetDefaults() ScreenshotOptions {
 	}
 	return s
 }
+
+//nolint:errcheck,gosec
```
Comment (yuri-tceretian) [inline pkg/services/screenshot/option.go:47]: I am a little bit concerned about this.

probably the better approach would be `_, _ = h.Write...`?

**Review by yuri-tceretian — APPROVED** [review]
LGTM

File: pkg/services/screenshot/option.go (hunk 1 above)
Comment (grobinson-grafana) [reply pkg/services/screenshot/option.go:47]: I did think that, but we've used nolint elsewhere for the same hash function. Will fix!

Comment (yuri-tceretian) [reply pkg/services/screenshot/option.go:47]: ah. ok then

**Pull request merged**